src/test/java/edu/harvard/iq/dataverse/export/DDIExporterTest.java

```diff
@@ -1,6 +1,6 @@
 package edu.harvard.iq.dataverse.export;
 
-import com.google.gson.Gson;
+import com.google.gson.*;
```

Comment:
For a long time we've been discouraging wildcard imports but there was some recent discussion on reconsidering this position. Nothing is written at https://guides.dataverse.org/en/5.10/developers/coding-style.html either way. 😄